tests/foreman/cli/test_repository_set.py

```diff
@@ -147,3 +149,118 @@ def test_repositoryset_available_repositories(self):
             sum(int(repo['enabled'] == u'true') for repo in result.stdout),
             0
         )
+
+    def test_repositoryset_enable_by_name(self):
+        """@Test: Enable repo from reposet by names of reposet, org and product
+
+        @Feature: Repository-set
+
+        @Assert: Repository was enabled
+
+        """
+        org = make_org()
+        manifest = manifests.clone()
+        upload_file(manifest, remote_file=manifest)
+        result = Subscription.upload({
+            u'file': manifest,
+            u'organization-id': org['id'],
+        })
+        self.assertEqual(result.return_code, 0)
+        result = RepositorySet.enable({
+            u'name': REPOSET['rhva6'],
+            u'organization': org['name'],
+            u'product': PRDS['rhel'],
+            u'releasever': '6Server',
+            u'basearch': 'x86_64',
+        })
+        self.assertEqual(result.return_code, 0)
+        result = RepositorySet.available_repositories({
+            u'name': REPOSET['rhva6'],
+            u'organization': org['name'],
+            u'product': PRDS['rhel'],
+        })
+        self.assertEqual(result.return_code, 0)
+        for repo in result.stdout:
+            if repo['release'] == '6Server' and repo['arch'] == 'x86_64':
+                enabled = repo['enabled']
```

Comment:
Approach with lambda seems more 'pythonic' as per different suggestions in the Internet and in the books, but, in general, it doesn't make serious difference